portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -21,8 +21,11 @@
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.List;
```

Comment:
I think the order of this and the one below should be swapped? Do you have a linter that will format for you?